### ISSUE TITLE
Improving Example Docs

### ISSUE THREAD

**Issue by ahuang11** [Unidata/siphon#225]
When I was scrolling through the examples, particularly the first example, I was a bit confused on how I could really apply the example because it just returns a list (wasn't sure what I could do with the list).

![image](https://user-images.githubusercontent.com/15331990/39556749-3e716cae-4e48-11e8-823c-1b98745bba61.png)

Thanks to John, I found their workshop tutorial which contained an extended version of the example and I found it to be much more useful.

https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Siphon/Siphon%20Tutorial.ipynb

tldr:
https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Siphon/Siphon%20Tutorial.ipynb and https://unidata.github.io/python-gallery/index.html should be linked somewhere because it has many applicable examples that the current example's page does not!

**Comment by dopplershift** [comment]
Linking to the workshop in general is a good idea. What would make the most sense though is to steal the ideas from the workshop notebook and incorporate them directly into a few examples.
